pkg/gadgets/seccomp/gadget.go

```diff
@@ -315,7 +315,7 @@ func (t *Trace) Generate(trace *gadgetv1alpha1.Trace) {
 			trace.Spec.Filter.ContainerName,
 		)
 		if mntns == 0 {
-			trace.Status.OperationError = fmt.Sprintf("Container %s/%s/%s not found on this node",
+			trace.Status.OperationWarning = fmt.Sprintf("Container %s/%s/%s not found on this node",
```

Comment:
nit: Please delete the "on this node". It does not provide any useful information and does not look that well on the CLI 🙂:

```
$ ./kubectl-gadget seccomp-advisor start -p non-existing-node
ftY5L0LLtZ8mUIsi

$ ./kubectl-gadget seccomp-advisor stop ftY5L0LLtZ8mUIsi
Failed to run the gadget on all nodes: Pod default/non-existing-node not found on this node
Error: Failed to run the gadget on all nodes: None of them succeeded
```